iocage/cli/provision.py

```diff
@@ -118,6 +119,7 @@ def _execute_provisioner(
     jail: 'iocage.lib.Jail.JailsGenerator'
 ) -> typing.Generator['iocage.lib.events.IocageEvent', None, None]:
     for event in jail.provisioner.provision():
-        if isinstance(event, iocage.lib.events.JailLaunch) and event.done:
-            print(event.stdout)
         yield event
+        if isinstance(event, iocage.lib.events.JailCommandExecution):
```

Comment:
This should just be an additional line with `\` as there isn't anything else to be done if the if statement below isn't `True`

Reply:
I don't get it. Could you please further explain or leave a code snippet?